CS3220LAB3/src/Lab3/Food.java

```diff
@@ -0,0 +1,65 @@
+package Lab3;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import Lab3.CreateFood;
+
+
+
+@WebServlet(loadOnStartup = 1, urlPatterns = { "/food" })
+public class Food extends HttpServlet {
+	public void init() {
+		// init the application scope to have pre-set values
+		List<CreateFood> entries = new ArrayList<>();
+		entries.add(new CreateFood(entries.size(), "Eric", "Hello","fff"));
+		entries.add(new CreateFood(entries.size(), "John", "Hi there","ffff"));
```

Comment:
Price isn't double and allowing you to put "hello" as price. Quite ... non-sense for price attribute.